packages/server/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "./../ts/index.d.ts"
   ],
   "compilerOptions": {
-    "types": ["mocha", "node"]
+    "types": ["mocha", "node"],
+    "importHelpers": true
```

Comment:
Just need to add `tslib` as a dependency b/c of this change

Reply:
Will do.